Cargo.toml

```diff
@@ -78,6 +78,9 @@ abi3-py38 = ["abi3-py39", "pyo3-build-config/abi3-py38", "pyo3-ffi/abi3-py38"]
 abi3-py39 = ["abi3-py310", "pyo3-build-config/abi3-py39", "pyo3-ffi/abi3-py39"]
 abi3-py310 = ["abi3", "pyo3-build-config/abi3-py310", "pyo3-ffi/abi3-py310"]
 
+# Generates `python3.dll` import libraries automatically.
+auto-abi3-import-lib = ["pyo3-build-config/python3-dll-a"]
```

Comment:
Sorry for bike shedding, but I think `generate-abi3-import-lib` matching the function name would make the construction easier to follow.

Reply:
This is fine with me.